lib/actions/adapterActions.js

```diff
@@ -1076,6 +1077,10 @@ export function openAdapter(comName, baudRate, sdBleApiVersion) {
  */
 export function openJlinkAdapter(comName, serialNumber) {
     return dispatch => {
+        if (os.platform() === 'darwin') {
+            logger.info('Note: Adapters with Segger JLink debug probe requires MSD to be disabled to function properly on OSX. ' +
+                'Please visit www.nordicsemi.com/nRFConnectOSXfix for further instructions.');
```

Comment:
MSD is also problematic on Linux. Could you update this, so that it also shows the notice on Linux?

It is a bit misleading that the URL contains "nRFConnectOSXfix", but I guess we can live with that. At least the text on the web site mentions both OSX and Linux.

Reply:
Also, please bump the alpha version number, so that we can more easily identify the different versions now during release testing.

Reply:
OK